### DISCUSSION TITLE
Panel shows weather icon with a moon instead of the sun

### DISCUSSION THREAD

**Discussion by Rob65** [Blackymas/NSPanel_HA_Blueprint#1024] [General]
I have 2 HA systems. Both have HASOS 10.5 , Home Assistant 2023.9.0, ESPHome 2023.8.3 and the v.3.4 NSPanel Blueprint.
Both panels show TFT version 3.4, ESPHome 3.4.1 and Blueprint 3.4

One panel shows the weather information with a sun or a moon (information retrieved from sun.sun ?) but the other panel always shows a moon, even when sun.sun is above_horizon

As said, both systems are updated to the same version and on both panels I installed a new ESPHome when it was available beginning of this week. Both panels have been flashed with the same eu_panel.tft

What could be wrong with my system that makes the panel do this ?
Is there anything I can test to see if this is something in the panel (firmware or tft file) or some setting in my HomeAssistant configuration ?

**Comment by edwardtfn** [comment]
What if you compare the state of the weather entity in both systems? Are both with the same state?

**Reply by edwardtfn** [reply]
Are they showing different icons on Home page or the the detailed weather pages? Or both?

**Comment by Rob65** [comment]
It's the weather icon on the home page.

Ah, that fixed it.
I somehow had the sun integration disabled on the system that always shows a moon.
Enabling it (and rebooting my panel) now shows a sun instead of the moon.

Of course I checked everything, or so I thought, but I did not notice that the sun service was disabled ...
